ansible/roles/invoker/files/docker-runc

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
maybe we can rename it to `runc-simulator` or something similar as at a quick glance it may look like it's the real `runc` ?

Reply:
Or this is what it should be used always according to https://github.com/apache/incubator-openwhisk/blob/master/core/invoker/src/main/scala/whisk/core/containerpool/docker/RuncClient.scala#L42 ?

Reply:
We can certainly rename on mount, but yes: This is what should be used according to the code itself.

It might after all be more worth to do what you suggested and detect a missing runc and fallback to docker.

Reply:
> It might after all be more worth to do what you suggested and detect a missing runc and fallback to docker.

and do this from within this script instead of the scala code ?

Reply:
That would be scala code. The added benefit would be, that the logs clearly show you're using docker vs. runc which would surface config errors in production where you really want to use runc and not scratch your head why pause/unpause get slow.

Reply:
👍

Reply:
so in that case I can turn my changes into a PR, and, if I'm reading the PR correctly and the `docker-runc` won't be added, things should work.

Reply:
Sure you can! Go for it 😬

Reply:
created PR https://github.com/apache/incubator-openwhisk/pull/2796